### ISSUE TITLE
[BUG] Permissions not applied correctly / can't list channels with bot user

### ISSUE THREAD

**Issue by der-eismann** [RocketChat/Rocket.Chat#9895]
### Description:

I currently try to write a small Shell "library" to send messages to Rocket.Chat instead of E-Mails, however I am unable to list all existing channels on the server with a new bot user I created.

### Server Setup Information:

* Version of Rocket.Chat Server: 0.61.0
* Operating System: Ubuntu 16.04
* Deployment Method(snap/docker/tar/etc): snap
* Number of Running Instances: 1

### Steps to Reproduce:

1. Create new user with role bot
2. Send request to https://<Rocket.Chat Server>/api/v1/channels.list
(3. Add more permissions)
(4. Restart snap)

### Expected behavior:

Getting a list of all public channels on the server

### Actual behavior:

Getting a list of all **joined** public channels

### Further research:

So at first I thought it's a permission issue, so I added permissions for `view-other-user-channels` and others - but still no success. Next I gave the bot group **every** permission available, still no success. So I added the bot to the Admin group, still no success. Only after I removed every other group so the bot user was only in the Admin group it would list all available channels.
But this is not acceptable since these are way too much permissions.

**Comment by karlprieb** [comment]
@MarcosSpessatto it's an api problem? Can you take a look on that?

**Comment by MarcosSpessatto** [comment]
@karlprieb, see #10009.
@der-eismann you need set `view-c-room` to be able to return all public channels.

**Comment by der-eismann** [comment]
Thanks for the quick fix! Can someone say if it will be merged into a hotfix? Or does it have to wait until 0.63?

**Comment by karlprieb** [comment]
Hey @der-eismann @rodrigok can answer that!

@MarcosSpessatto thanks for the fix :D
